frontend/app/routes/$lang+/_public+/apply+/$id+/adult-child/children/index.tsx

```diff
@@ -93,43 +139,68 @@ export default function ApplyFlowChildSummary() {
         <Progress aria-labelledby="progress-label" value={40} size="lg" />
       </div>
       <div className="max-w-prose">
-        <fetcher.Form method="post" aria-describedby="form-instructions-sin form-instructions" noValidate>
+        <fetcher.Form method="post" onSubmit={handleSubmit} aria-describedby="form-instructions-sin form-instructions">
```

Comment:
Remove the `aria-describedby` on the form.

https://www.w3.org/WAI/tutorials/forms/instructions/
> Provide such instructions before the `form` element to ensure that it is read aloud by screen readers before they switch to “Forms Mode”.